### PR TITLE
Bug Report Template

### LINKED ISSUE #17: Template for bug reports
**Behavior of the program**
Please write a clear and concise description of what the bug is.

**Expected behaviour**
Please describe clear and concise what you expected to happen.

**To Reproduce**
Please add all steps to reproduce the behaviour:

**Screenshots**
If applicable, please add screenshots to help explain your problem. 

**Additional context**

### PR DESCRIPTION
**Pull request by EggY0lk**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+JRE Version: [Insert JRE Version here, a clear screenshot works too]
+
+/* To check JRE Version, open command prompt/terminal and type *java -version* */
+
+**Additional context**
+Add any other context about the problem here.
+
+
+Issue Template modified from FreeTube. 
+[https://github.com/FreeTubeApp/FreeTube]
```
Comment (LukeHuckman) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Bruh, I was just joking about this.

Comment (EggY0lk) [reply .github/ISSUE_TEMPLATE/bug_report.md]: bruh, too late. Well at least we wont get dmca or whatever like a certain other repo.

Comment (LukeHuckman) [reply .github/ISSUE_TEMPLATE/bug_report.md]: The licence covers the code itself. Templates can be shared lmao. They're called templates for a reason.

Comment (EggY0lk) [reply .github/ISSUE_TEMPLATE/bug_report.md]: so wht now.  no harm in adding it.

Comment (LukeHuckman) [reply .github/ISSUE_TEMPLATE/bug_report.md]: It's not necessary.

Comment (EggY0lk) [reply .github/ISSUE_TEMPLATE/bug_report.md]: ok editing it now

**Pull request merged**